apiserver/uniter/uniter.go

```diff
@@ -145,14 +152,25 @@ func NewUniterAPI(st *state.State, resources facade.Resources, authorizer facade
 	}, nil
 }
 
+// NewUniterAPIV5 creates an instance of the V5 uniter API.
+func NewUniterAPIV5(st *state.State, resources facade.Resources, authorizer facade.Authorizer) (*UniterAPIV5, error) {
+	uniterAPI, err := NewUniterAPI(st, resources, authorizer)
+	if err != nil {
+		return nil, err
+	}
+	return &UniterAPIV5{
+		UniterAPI: *uniterAPI,
+	}, nil
+}
+
 // NewUniterAPIV4 creates an instance of the V4 uniter API.
 func NewUniterAPIV4(st *state.State, resources facade.Resources, authorizer facade.Authorizer) (*UniterAPIV4, error) {
-	uniterAPI, err := NewUniterAPI(st, resources, authorizer)
+	uniterAPI, err := NewUniterAPIV5(st, resources, authorizer)
```

Comment:
It's what was happening before anyway (there was an invisible V5 at the end of `NewUniterAPI` that became visible when V6 sprang into existence). The differences between the API versions are cumulative - if we just embedded `UniterAPI` in `UniterAPIV4` we'd need to copy the V5 methods into it as well. 

We could do it the other way around, implementing V5 in terms of V4 and V6 embedding V5, but it seems clearer to have the current version be the default case and the older (hopefully smaller) ones the deltas.